Analysis.ipynb

```diff
@@ -0,0 +1,1186 @@
+{
```

Comment:
The best mode should be:
best_model = make_pipeline(preprocessor, LogisticRegression(random_state=123, class_weight= 'balanced', C= 0.000002, max_iter=500))